demisto_sdk/commands/lint/linter.py

```diff
@@ -287,6 +293,37 @@ def run_bandit(self, py_num) -> int:
                 self.lock.release()
             return 1
 
+    def run_vulture(self, py_num) -> int:
+        """Run vulture
+
+        Args:
+            py_num: The python version in use
+
+        Returns:
+            int. 0 on successful vulture run, 1 otherwise.
+        """
+        lint_files = self._get_lint_files()
+        python_exe = 'python2' if py_num < 3 else 'python3'
+        cmd_args = [python_exe, '-m', 'vulture', lint_files, '--min-confidence 100']
```

Comment:
Let's get the confidence level from an environment variable (with a default of 100). That way if a developer wants to run it with a lower confidence level she can simply set an environment variable.